blog/migrations/0001_initial.py

```diff
@@ -1,9 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.11.1 on 2017-05-27 17:45
```

Comment:
You should not touch autogenerated files. Configure linter to skip checking them.